modules/external-api/src/main/java/org/opencastproject/external/common/ApiResponses.java

```diff
@@ -82,9 +82,25 @@ public static Response serverError(String message, Object... args) {
      * @return The new {@link Response}
      */
     public static Response ok(ApiVersion version, String body) {
+      // FIXME: This does not make sense. Sending content type 'application/json' for a plain string?
```

Comment:
Well, actually it should not return plain strings at all since it "officially" only accepts application/json, application/v1.0.0+json and application/v1.1.0+json...

That is, however, not scope of this PR. I've created https://opencast.jira.com/browse/MH-12798 for this. Could you please mention MH-12798 in the comment?

Maybe something like: "// MH-12798: The External API should consistently return JSON strings"